src/e2e/java/uk/gov/hmcts/reform/tools/builders/Appeal.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.tools.builders;
+
+import java.io.IOException;
+import java.util.*;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import uk.gov.hmcts.reform.tools.utils.*;
+
+public class Appeal {
```

Comment:
Not too fussed either way but might want to use `Lombok` here to avoid having to import getters and setters and to keep consistent with all other domain classes within Case Loader